knowledge_base/python_wheel_poetry/python_wheel_poetry/hello.py

```diff
@@ -0,0 +1,12 @@
+"""
```

Comment:
Shouldn't this be in `src/` for uniformity?

Reply:
Done.